sdk/frontdoor/Azure.ResourceManager.FrontDoor/api/Azure.ResourceManager.FrontDoor.netstandard2.0.cs

```diff
@@ -430,90 +385,18 @@ public partial class CacheConfiguration
         public string QueryParameters { get { throw null; } set { } }
         public Azure.ResourceManager.FrontDoor.Models.FrontDoorQuery? QueryParameterStripDirective { get { throw null; } set { } }
     }
-    public partial class CheckNameAvailabilityInput
-    {
-        public CheckNameAvailabilityInput(string name, Azure.ResourceManager.FrontDoor.Models.ResourceType resourceType) { }
-        public string Name { get { throw null; } }
-        public Azure.ResourceManager.FrontDoor.Models.ResourceType ResourceType { get { throw null; } }
-    }
-    public partial class CheckNameAvailabilityOutput
-    {
-        internal CheckNameAvailabilityOutput() { }
-        public string Message { get { throw null; } }
-        public Azure.ResourceManager.FrontDoor.Models.Availability? NameAvailability { get { throw null; } }
-        public string Reason { get { throw null; } }
-    }
     public partial class CustomHttpsConfiguration
     {
-        public CustomHttpsConfiguration(Azure.ResourceManager.FrontDoor.Models.FrontDoorCertificateSource certificateSource, Azure.ResourceManager.FrontDoor.Models.FrontDoorTlsProtocolType protocolType, Azure.ResourceManager.FrontDoor.Models.MinimumTLSVersion minimumTlsVersion) { }
+        public CustomHttpsConfiguration(Azure.ResourceManager.FrontDoor.Models.FrontDoorCertificateSource certificateSource, Azure.ResourceManager.FrontDoor.Models.FrontDoorTlsProtocolType protocolType, Azure.ResourceManager.FrontDoor.Models.MinimumTlsVersion minimumTlsVersion) { }
         public Azure.ResourceManager.FrontDoor.Models.FrontDoorCertificateSource CertificateSource { get { throw null; } set { } }
-        public Azure.ResourceManager.FrontDoor.Models.FrontDoorCertificateType? CertificateType { get { throw null; } set { } }
-        public Azure.ResourceManager.FrontDoor.Models.MinimumTLSVersion MinimumTlsVersion { get { throw null; } set { } }
+        public Azure.ResourceManager.FrontDoor.Models.FrontDoorEndpointConnectionCertificateType? CertificateType { get { throw null; } set { } }
+        public Azure.ResourceManager.FrontDoor.Models.MinimumTlsVersion MinimumTlsVersion { get { throw null; } set { } }
```

Comment:
A lot of other RPs have this enum type `MinimumTlsVersion`, please consider to prepend a prefix for this

Reply:
Changed to `FrontDoorRequiredMinimumTlsVersion`